src/main/java/uk/ac/cam/cl/dtg/teaching/pottery/containers/AttachListener.java

```diff
@@ -24,15 +24,19 @@
 
   private final StringBuilder output = new StringBuilder();
   private boolean closed = false;
+  private final int outputLimit;
+
+  AttachListener(int outputLimit) {
```

Comment:
Could you add some units to the variable name for future reference?

Reply:
Will do once decided!

Reply:
Done.